### DISCUSSION TITLE
automatic import of module

### DISCUSSION THREAD

**Discussion by aurel1sergent** [YetiForceCompany/YetiForceCRM#17372] [How to]
hello, actually using V 6.4 of yetiforce, we often need to replicate some modules which are created in one environment to another. 
we are doing this using the GUI : 

- create module
- export using the button functionality
- login on the other environment and import the package previously created using the install button. 

is there a way we could automate this task instead of using the GUI webapp ? 
regards

**Comment by rskrzypczak** [comment]
create your own script and use ` \vtlib\PackageExport, \vtlib\PackageImport()` functions
